index.js

```diff
@@ -0,0 +1 @@
+// I have choosed the api: https://www.themealdb.com/api/json/v1/1/categories.php
```

Comment:
hi @abenezer-addisu 
✔️  good  implementation in adding the api as per the project requirement.
there is nothing more to say other than its time to move to the next task 🎉 .

# 👍 👍 👍